pull-request-builder-general-availability.rst

```diff
@@ -0,0 +1,59 @@
+.. post:: Jan 8, 2021
+   :tags: pr-builder, continuous-documentation, feature
+   :author: Eric
+   :location: BND
+
+Pull Request Builders available for all users
+=============================================
+
+We're excited to announce that `Pull Request building <https://docs.readthedocs.io/en/latest/guides/autobuild-docs-for-pull-requests.html>`_ is now available for all Read the Docs users.
+We have been working on this feature for over a year,
+and having it available for all our users is a major milestone.
+
+This feature allows users to confirm documentation builds correctly for all of their commits,
+not just ones merged into branches that are activated on Read the Docs.
+This moves documentation into your continuous integration pipeline,
+and improves the workflow for everyone working on documentation.
+
+.. figure:: img/github-build-status-reporting.gif
+    :alt: GitHub Build Status Reporting for Pull Requests.
+    :width: 70%
+
+    GitHub Build Status Reporting for Pull Requests
+
+You can read more about this new feature in our :doc:`previous blog post announcing it </building-docs-for-pull-requests>` and in our `documentation for the feature <https://docs.readthedocs.io/en/latest/guides/autobuild-docs-for-pull-requests.html>`_.
```

Comment:
```suggestion
You can read more about this new feature in our :doc:`previous blog post announcing it </building-docs-for-pull-requests>` and in our `documentation for the feature <https://docs.readthedocs.io/page/guides/autobuild-docs-for-pull-requests.html>`_.
```

Reply:
I have updated the docs for pull requests here https://github.com/readthedocs/readthedocs.org/pull/7823